internal/cmd/update-submodule-versions/main_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const sampleGoMod = `module github.com/riverqueue/river
 
-go 1.21.4
```

Comment:
Oops, didn't mean to change this line, but I'm not sure it matters?

Reply:
Won't matter as far as the test is concerned.